src/components/player/NowPlayingInfoView.tsx

```diff
@@ -128,12 +129,12 @@ const NowPlayingInfoView = () => {
   }
 
   return (
-    <Animated
-      animationInDuration={500}
-      animationOutDuration={0}
-      animationIn="fadeIn"
-      animationOut="fadeOut"
-      isVisible={!isLoading}
+    <CSSTransition
+      timeout={{
+        enter: 500,
+        exit: 0,
+      }}
+      classNames="grid-animated"
     >
       {currentArtist && playQueue.entry?.length > 0 && (
```

Comment:
Getting an error here: `React.Children.only expected to receive a single React element child.`

Seems like you need to wrap this block in another fragment `<></>`